src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -805,6 +805,8 @@ private function startClass($class, $baseClass, $namespace)
  *
  * This class has been auto-generated
  * by the Symfony Dependency Injection Component.
+ *
+ * @final since Symfony 3.3
```

Comment:
As far as #20493 is concerned, this shouldn't matter to me as DebugClassLoader should be able to deal with both situations. :)